x-pack/plugins/fleet/common/settings/agent_policy_settings.ts

```diff
@@ -126,4 +128,18 @@ export const AGENT_POLICY_ADVANCED_SETTINGS: SettingsConfig[] = [
       })
       .default({}),
   },
+  {
+    name: 'agent.logging.level',
+    hidden: true,
+    title: i18n.translate('xpack.fleet.settings.agentPolicyAdvanced.agentLoggingLevel', {
+      defaultMessage: 'Agent Logging Level',
+    }),
+    description: i18n.translate('xpack.fleet.settings.agentPolicyAdvanced.goMaxProcsDescription', {
+      defaultMessage: 'Set the Agent log level. The default log level is "info".',
+    }),
+    api_field: {
+      name: 'agent_logging_level',
+    },
+    schema: z.nativeEnum(agentLoggingLevels),
```

Comment:
Should we set `info` as default on UI? Or if it's not set by default, that's good too.